acceptance/go.mod

```diff
@@ -29,6 +29,11 @@ require (
 	sigs.k8s.io/gateway-api v0.7.1
 )
 
+replace (
+	github.com/hashicorp/consul-k8s/control-plane => ../control-plane
```

Comment:
Should this remain this way long term?

Reply:
honestly I'm not sure, the acceptance tests should be pulling in the latest version of the code they're working with when testing, as of now if we add a new type to the control-plane we have to publish that first and then pull in the new version here, so the acceptance tests will always have to lag a bit against what is published